WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 269-507-995
ERROR demo.py:12 error msg
WARNING demo.py:13 warn msg
INFO demo.py:14 info msg
DEBUG demo.py:15 debug msg
INFO _internal.py:88 127.0.0.1 - - [20/Feb/2019 21:59:00] "GET /api/v1.0/index HTTP/1.1" 200 -
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 269-507-995
ERROR demo.py:12 error msg
WARNING demo.py:13 warn msg
INFO demo.py:14 info msg
DEBUG demo.py:15 debug msg
INFO _internal.py:88 127.0.0.1 - - [20/Feb/2019 21:59:42] "GET /api/v1.0/index HTTP/1.1" 200 -
